macros/InterfaceData.json

```diff
@@ -1275,6 +1287,14 @@
     "inh"  : "Event",
     "impl" : []
   },
+  "RTCIdentityProviderGlobalScope": {
+    "inh"  : "WorkerGlobalScope",
+    "impl" : []
+  },
+  "RTCErrorEvent": {
```

Comment:
Which? RTCErrorEvent? This probably will be implemented eventually but you're right, it isn't now, so I'll remove it. Thanks!